src/lib/channels.py

```diff
@@ -0,0 +1,116 @@
+import time
+
+from core_data_modules.cleaners import Codes
+from core_data_modules.traced_data import Metadata
+from dateutil.parser import isoparse
+
+from src.lib import PipelineConfiguration
+
+class Channels(object):
```

Comment:
Can you submit this file as a separate PR? This is so we can additionally have someone from RDA as a reviewer of the timestamps

Reply:
Okay will do thanks

Reply:
Remove this file from here before merging or, if it breaks the pipeline, delete the timestamps.

(This is because I want the timestamps to be introduced in PR https://github.com/AfricasVoices/Project-ICRAF/pull/11)

Reply:
File deleted